main/res/raw/changelog_master.md

```diff
@@ -0,0 +1,13 @@
+## Nightly build:
+
+### Map:
```

Comment:
But I would follow Codacys suggestion to remove the `:` behind headings

Reply:
Ah, forgot that. Have updated PR accordingly...